syncs/github-issue-comments/main.ts

```diff
@@ -41,12 +42,14 @@ const octokit = new OctokitWithThrottling({
       },
 });
 const commentsBuffer = [];
+const perPage = params.perPage || 100
 
 const iterator = octokit.paginate.iterator(`GET /repos/${owner}/${repo}/issues/comments`, {
     headers: {
         'X-GitHub-Api-Version': '2022-11-28'
     },
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```

Comment:
See: https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments-for-a-repository--parameters